packages/editor/src/components/post-author/style.scss

```diff
@@ -0,0 +1,94 @@
+.components-post-author__autocomplete__wrapper {
+	position: relative;
+}
+
+.components-post-author__autocomplete__hint,
+.components-post-author__autocomplete__input {
+	-webkit-appearance: none;
+	border: 2px solid;
+	border-radius: 0; /* Safari 10 on iOS adds implicit border rounding. */
+	box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	-webkit-box-sizing: border-box;
```

Comment:
We use autoprefixer to handle necessary prefixes automatically, so they should be removed from the source files altogether. In any case, `box-sizing` is redundant here as it's already being set to `border-box` for all elements.